### ISSUE TITLE
Wrong generation of a Collection type in swagger.json

### ISSUE THREAD

**Issue by fpotera** [stoicflame/enunciate#579]
Hello. 
We use enunciate to generate api for below class:

```java
import java.util.Collection;

public class CollectionResult<T extends Object>
{
	private Long count;
	private Collection<T> results;

	public CollectionResult()
	{
	}

	public CollectionResult(Collection<T> results, Long count)
	{
		this.results = results;
		this.count = count;
	}

	public void setCount(Long count)
	{
		this.count = count;
	}

	public void setResults(Collection<T> results)
	{
		this.results = results;
	}

	public Collection<T> getResults()
	{
		return results;
	}

	public Long getCount()
	{
		return count;
	}
}
```

It's wrong generated as:

```json
    "json_CollectionResult" : {
      "type" : "object",
      "title" : "CollectionResult",
          "properties" : {
            "results" : {
"description" : "",
"type" : "array",
"items" : {
"type" : "array",
"items" : {
  "type" : "file"
}
}
            },
            "count" : {
"description" : "",
"type" : "number"
            }
          }
```

Why it's generated as "array" in "array" of "file" ?

I think it should be just "array".

Am I right?

**Comment by stoicflame** [comment]
Fixed in 2.9.

**Comment by stoicflame** [comment]
[Enunciate 2.9.0 has been released.](https://github.com/stoicflame/enunciate/releases/tag/v2.9.0)
